cmake/Modules/FindXGen.cmake

```diff
@@ -38,52 +38,27 @@
 if(APPLE)
   find_path(MAYA_BASE_DIR ../../devkit/include/maya/MFn.h PATH
     ENV MAYA_LOCATION
-    "/Applications/Autodesk/maya2016/Maya.app/Contents"
-    "/Applications/Autodesk/maya2015/Maya.app/Contents"
-    "/Applications/Autodesk/maya2014/Maya.app/Contents"
-    "/Applications/Autodesk/maya2013/Maya.app/Contents"
-    "/Applications/Autodesk/maya2012.17/Maya.app/Contents"
-    "/Applications/Autodesk/maya2012/Maya.app/Contents"
-    "/Applications/Autodesk/maya2011/Maya.app/Contents"
-    "/Applications/Autodesk/maya2010/Maya.app/Contents"
-  )
+    "/Applications/Autodesk/maya2019/Maya.app/Contents"
+    "/Applications/Autodesk/maya2018/Maya.app/Contents"
+    "/Applications/Autodesk/maya2017/Maya.app/Contents"
```

Comment:
Should we keep Maya 2017 here?

Reply:
For Linux and MacOS we may keep it, VS2012 deprecation affects only Windows.
Maya 2017 & 2018 sdk for Linux & MacOS use gcc 4.8.1 and Xcode 6.1 or Intel Composer XE 2015 Update 1.